### PR TITLE
python.pkgs.applicationinsights: init at 0.11.9

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
Add more Azure support into Nixos packages.

Not related to #60435 (separate pypi package and repo)

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
$ nix path-info -Sh result/
/nix/store/hgyfrsfw3my1p14lbsmdz02ynmqz7cc0-python3.5-applicationinsights-0.11.9          95.0M

### CODE REVIEW

File: pkgs/development/python-modules/applicationinsights/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hyjdv6xnswgqvip8y164piwfach9hjkbp7vc2qzhd7amjpim89h";
+  };
+
+  propagatedBuildInputs = [ portalocker ];
+
+  meta = with stdenv.lib; {
+    description = "This project extends the Application Insights API surface to support Python.";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/applicationinsights/default.nix]:
```suggestion
    description = "This project extends the Application Insights API surface to support Python";
```

File: pkgs/development/python-modules/applicationinsights/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    homepage = https://github.com/Microsoft/ApplicationInsights-Python;
+    license = licenses.mit;
+    maintainers = with maintainers; [ jonringer ];
+    platforms = platforms.unix;
```
Comment (worldofpeace) [inline pkgs/development/python-modules/applicationinsights/default.nix]:
```suggestion
```

Don't see why we'd need to set platforms when `buildPythonPackage` already does this.
Was there an intention to restrict it?

File: pkgs/development/python-modules/portalocker/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    sphinx
+    flake8
+    pytest
+    pytestcov
+    pytest-flakes
+    pytestpep8
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Portalocker is a library to provide an easy API to file locking.";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/portalocker/default.nix]:
```suggestion
    description = "Portalocker is a library to provide an easy API to file locking";
```

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build python37.pkgs.applicationinsights python36.pkgs.applicationinsights python35.pkgs.applicationinsights python.pkgs.applicationinsights pypy.pkgs.applicationinsights python37.pkgs.portalocker python36.pkgs.portalocker python35.pkgs.portalocker python.pkgs.portalocker
pypy.pkgs.portalocker

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Please call packages from `pkgs/top-level/python-packages.nix`

File: pkgs/development/python-modules/applicationinsights/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/development/python-modules/applicationinsights/default.nix]: Don't remember the original reasoning, ill remove it.

File: pkgs/development/python-modules/applicationinsights/default.nix (hunk 1 above)
Comment (jonringer) [reply pkgs/development/python-modules/applicationinsights/default.nix]: Applied

File: pkgs/development/python-modules/portalocker/default.nix (hunk 3 above)
Comment (jonringer) [reply pkgs/development/python-modules/portalocker/default.nix]: Applied

**Comment by jonringer** [comment]
> Please call packages from `pkgs/top-level/python-packages.nix`

Yea, got used to doing version bumps/patches forgot about adding the top-level entries, all fixed now :) @worldofpeace

**Comment by worldofpeace** [comment]
> > Please call packages from `pkgs/top-level/python-packages.nix`
> 
> Yea, got used to doing version bumps/patches forgot about adding the top-level entries, all fixed now :) @worldofpeace

Thanks, let's see if it builds everywhere :+1: 

@GrahamcOfBorg build python37.pkgs.applicationinsights python36.pkgs.applicationinsights python35.pkgs.applicationinsights python.pkgs.applicationinsights pypy.pkgs.applicationinsights python37.pkgs.portalocker python36.pkgs.portalocker python35.pkgs.portalocker python.pkgs.portalocker
pypy.pkgs.portalocker

**Comment by worldofpeace** [comment]
Thanks @jonringer for contributing :cherry_blossom:

**Pull request merged**